instrumentation/view-click/src/main/kotlin/io/opentelemetry/android/instrumentation/view/click/internal/ViewUtils.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.android.instrumentation.view.click.internal
+
+import io.opentelemetry.api.common.AttributeKey
+import io.opentelemetry.api.common.AttributeKey.doubleKey
+import io.opentelemetry.api.common.AttributeKey.longKey
+import io.opentelemetry.api.common.AttributeKey.stringKey
+
+const val APP_SCREEN_CLICK_EVENT_NAME = "app.screen.click"
+const val VIEW_CLICK_EVENT_NAME = "event.app.widget.click"
+val viewNameAttr: AttributeKey<String> = stringKey("app.widget.name")
+
+val xCoordinateAttr: AttributeKey<Double> = doubleKey("app.screen.coordinate.x")
+val yCoordinateAttr: AttributeKey<Double> = doubleKey("app.screen.coordinate.y")
```

Comment:
These should not be `Double` they should be `Long`. It is [specified](https://github.com/open-telemetry/semantic-conventions/tree/2073d8ab5e6274e6f883b8ff89d358fdee510c58/docs/app#click-events) in pixels, which is never fractional/decimal and is always a whole number.

Reply:
They're floats in the platform between.

Reply:
Ok, I read these docs https://developer.android.com/reference/android/view/MotionEvent#getX(int) and it sounds like there is the possibility of some devices providing subpixel precision, in which case it sounds like we should probably [change the current semconv](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/app.md) to match. 

What do you think?

Reply:
Looks like you changed it to long..thanks!